macros/scripts/text.asm

```diff
@@ -136,3 +136,10 @@ text_far: MACRO
 	dw \1
 	db BANK(\1)
 ENDM
+
+	enum_start $50
```

Comment:
Either use `enum_set` or remove `enum_set` from the macros.
Maybe it'd be a good idea to remove this line and test the game. If nothing breaks, this commit is correct ;)

Reply:
Nothing I've tested is breaking without it. :)